### ISSUE TITLE
PRETRAIN_PATH在哪里用到

### ISSUE THREAD

**Issue by guanyonglai** [JDAI-CV/fast-reid#101]
我写了PRETRAIN_PATH但是好像没有用到。我这边训练market1501上top1只有93.6，不知道是不是没有载入预训练模型的原因

**Comment by guanyonglai** [comment]
我在imgnet官网下载的resnet50_ibn_a.pth.tar模型和本代码的不匹配，你们的预训练模型都在哪里下的呢

**Comment by L1aoXingyu** [comment]
请把你的配置文件发出来

**Comment by guanyonglai** [comment]
后来加了预训练模型，但是top1也只有94.6呀，没作者说的那么高，

CUDNN_BENCHMARK: true
DATALOADER: {NUM_INSTANCE: 16, NUM_WORKERS: 16, PK_SAMPLER: true}
DATASETS:
  COMBINEALL: false
  NAMES: [Market1501]
  TESTS: [Market1501]
INPUT:
  DO_AUGMIX: false
  DO_AUTOAUG: true
  DO_CJ: false
  DO_FLIP: true
  DO_PAD: true
  FLIP_PROB: 0.5
  PADDING: 10
  PADDING_MODE: constant
  REA:
    ENABLED: true
    MEAN: [123.675, 116.28, 103.53]
    PROB: 0.5
  RPT: {ENABLED: false, PROB: 0.5}
  SIZE_TEST: [384, 128]
  SIZE_TRAIN: [384, 128]
MODEL:
  BACKBONE: {DEPTH: 50, LAST_STRIDE: 1, NAME: build_resnet_backbone, NORM: BN, NORM_SPLIT: 1,
    PRETRAIN: true, PRETRAIN_PATH: /home/guanyonglai/.cache/torch/checkpoints/resnet50_ibn_a.pth.tar,
    WITH_IBN: true, WITH_NL: false, WITH_SE: false}
  HEADS: {CLS_LAYER: circle, IN_FEAT: 256, MARGIN: 0.35, NAME: BNneckHead, NECK_FEAT: after,
    NORM: BN, NORM_SPLIT: 1, NUM_CLASSES: 751, POOL_LAYER: gempool, REDUCTION_DIM: 512,
    SCALE: 64}
  LOSSES:
    CE: {ALPHA: 0.3, EPSILON: 0.1, SCALE: 0.125}
    FL: {ALPHA: 0.25, GAMMA: 2, SCALE: 1.0}
    NAME: [CrossEntropyLoss, TripletLoss]
    TRI: {HARD_MINING: true, MARGIN: 0.0, NORM_FEAT: false, SCALE: 0.2, USE_COSINE_DIST: false}
  META_ARCHITECTURE: MGN
  OPEN_LAYERS: [b1_pool, b1_head, b2_pool, b2_head, b21_pool, b21_head, b22_pool,
    b22_head, b3_pool, b3_head, b31_pool, b31_head, b32_pool, b32_head, b33_pool,
    b33_head]
  PIXEL_MEAN: [123.675, 116.28, 103.53]
  PIXEL_STD: [58.395, 57.120000000000005, 57.375]
  WEIGHTS: ''
OUTPUT_DIR: logs/market/mgn_R50-ibn
SOLVER:
  BASE_LR: 0.00035
  BIAS_LR_FACTOR: 2.0
  CHECKPOINT_PERIOD: 6000
  DELAY_ITERS: 9000
  ETA_MIN_LR: 7.7e-07
  FREEZE_ITERS: 2000
  GAMMA: 0.1
  HEADS_LR_FACTOR: 1.0
  IMS_PER_BATCH: 64
  LOG_PERIOD: 200
  MAX_ITER: 18000
  MOMENTUM: 0.9
  OPT: Adam
  SCHED: DelayedCosineAnnealingLR
  STEPS: [30, 55]
  SWA: {ENABLED: false, ETA_MIN_LR: 3.5e-06, ITER: 0, LR_FACTOR: 10.0, LR_SCHED: false,
    PERIOD: 10}
  WARMUP_FACTOR: 0.01
  WARMUP_ITERS: 2000
  WARMUP_METHOD: linear
  WEIGHT_DECAY: 0.0005
  WEIGHT_DECAY_BIAS: 0.0
TEST:
  AQE: {ALPHA: 3.0, ENABLED: false, QE_K: 5, QE_TIME: 1}
  EVAL_PERIOD: 2000
  IMS_PER_BATCH: 256
  METRIC: cosine
  PRECISE_BN: {DATASET: DukeMTMC, ENABLED: false, NUM_ITER: 300}
  RERANK: {ENABLED: false, K1: 20, K2: 6, LAMBDA: 0.3}

**Comment by L1aoXingyu** [comment]
你把 `WITH_NL: True` 就可以复现了

**Comment by guanyonglai** [comment]
你好，把WITH_NL改成True效果也达不到呀
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: Rank-1
[06/12 22:46:24] fastreid.evaluation.testing INFO: 94.5%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: Rank-5
[06/12 22:46:24] fastreid.evaluation.testing INFO: 98.1%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: Rank-10
[06/12 22:46:24] fastreid.evaluation.testing INFO: 98.9%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: mAP
[06/12 22:46:24] fastreid.evaluation.testing INFO: 85.0%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: mINP
[06/12 22:46:24] fastreid.evaluation.testing INFO: 53.8%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: TPR@FPR=0.0001
[06/12 22:46:24] fastreid.evaluation.testing INFO: 5.2%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: TPR@FPR=0.001
[06/12 22:46:24] fastreid.evaluation.testing INFO: 28.4%
[06/12 22:46:24] fastreid.evaluation.testing INFO: Task: TPR@FPR=0.01
[06/12 22:46:24] fastreid.evaluation.testing INFO: 94.5%

**Comment by L1aoXingyu** [comment]
@guanyonglai 
如果你是多卡训练，需要将 `backbone` 和 `head` 中的 `BN` 替换为 `syncBN`
